spec/models/manageiq/providers/redhat/infra_manager/refresher_3_0_spec.rb

```diff
@@ -246,7 +246,7 @@ def assert_specific_vm_powered_on
       :standby_action        => nil,
       :connection_state      => "connected",
       :cpu_affinity          => nil,
-      :memory_reserve        => nil,
+      :memory_reserve        => 715_128_832,
```

Comment:
@Fryguy this column may be another candidate for rename to `memory_reserve_mb`.  It appears 1024 megabytes is stored as `1024`.